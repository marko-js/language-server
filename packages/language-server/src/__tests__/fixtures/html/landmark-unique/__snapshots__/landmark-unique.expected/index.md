## Diagnostics
### Ln 1, Col 2
```marko
> 1 | <div role="region">Region 1</div>
    |  ^^^ Fix any of the following:
  The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable
  2 | <div role="region">Region 2</div>
  3 |
```

