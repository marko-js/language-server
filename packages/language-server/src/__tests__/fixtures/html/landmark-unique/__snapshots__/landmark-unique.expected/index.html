<div data-marko-node-id="0" role="region">Region 1</div><div data-marko-node-id="1" role="region">Region 2</div>